CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Added
 * [#5654](https://github.com/thanos-io/thanos/pull/5654) Query: add `--grpc-compression` flag that controls the compression used in gRPC client. With the flag it is now possible to compress the traffic between Query and StoreAPI nodes - you get lower network usage in exchange for a bit higher CPU/RAM usage.
+- [#5561](https://github.com/thanos-io/thanos/pull/5650) Query Frontend: Add sharded queries metrics.
```

Comment:
Should be `#5650` rather than `#5561`. That's the last thing then we can merge :)